packages/lib/src/smartthings-command.ts

```diff
@@ -56,24 +84,39 @@ export abstract class SmartThingsCommand extends Command implements SmartThingsC
 
 	private _profileName?: string
 
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	private _profileConfig?: { [name: string]: any }
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	get profileConfig(): { [name: string]: any } {
-		if (!this._profileConfig) {
+	private _profile?: Profile
+	get profile(): Profile {
+		if (!this._profile) {
 			throw new Error('SmartThingsCommand not properly initialized')
 		}
-		return this._profileConfig
+		return this._profile
 	}
 
+	cliConfig!: CLIConfig
+
 	stringConfigValue(keyName: string): string | undefined {
-		if (keyName in this.profileConfig) {
-			if (typeof this.profileConfig[keyName] === 'string') {
-				return this.profileConfig[keyName]
+		if (keyName in this.profile) {
+			const configValue = this.profile[keyName]
+			if (typeof configValue === 'string') {
+				return configValue
+			}
+			this.logger.warn(`expected string value for config key ${keyName} but got ${typeof this.profile[keyName]}`)
+			return undefined
+		}
+		this.logger.trace(`key ${keyName} not found in ${this.profileName} config`)
+	}
+
+	booleanConfigValue(keyName: string, defaultValue = false): boolean {
```

Comment:
It would be useful if all of the config access functions supported a default fallback value.